vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -911,6 +910,8 @@ static void display_entry_thread(EmuEnvState &emuenv) {
         SceGxmSyncObject *old_sync = display_callback->old_sync.get(emuenv.mem);
         SceGxmSyncObject *new_sync = display_callback->new_sync.get(emuenv.mem);
 
+        // sceGxmDisplayQueueAddEntry waits for both buffers to complete
+        renderer::wishlist(old_sync, display_callback->old_sync_timestamp);
         renderer::wishlist(new_sync, display_callback->new_sync_timestamp);
```

Comment:
You can add a check to perform the second wishlist only if old_sync != new_sync

Reply:
Okay. I didn’t add a comparative statement since I believed there was no functional difference between the two, however, considering it involves access to a lock, there may be a slight performance difference.